src/pages/HowToUse/index.js

```diff
@@ -19,7 +37,30 @@ const HowToUse = () => {
           />
         </Container>
       </Box>
-      <TagGeneratorInstructions />
+      <Box className='containerGray'>
+        <Container>
+          <Grid container className={classes.gridContainer}>
+            <Grid item xs={12} sm={10}>
+              <Typography variant='body1' className={classes.typoStyle}>
+                1. Under your project’s repository, click <SettingsGearIcon /> to paste your tags.
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={10} style={{ padding:'24px 16px 0' }}>
+              <Typography variant='body1' className={classes.typoStyle}>If you don’t see the <SettingsGearIcon />  button it means you don’t have &quot;edit repository settings&quot; privileges (and can’t perform the steps below).
+              </Typography>
+            </Grid>
+            <ImageComponent src='/images/instructions-step1.svg' alt='Step 1' />
+            <Grid>
+              <Typography variant='body1' className={classes.typoStyle} >2. Under <Box component="span" style={{ fontWeight: '700' }}>&quot;Topics&quot;</Box>, paste the topic you want to add to your repository. </Typography>
+            </Grid>
+            <ImageComponent src='/images/instructions-step2.svg' alt='Step 2' />
+            <Grid>
+              <Typography variant='body1' className={classes.typoStyle} >3. After you have finished adding your tags, click <Box component="span" style={{ fontWeight: '700' }}>Save Changes</Box> </Typography>
+            </Grid>
+            <ImageComponent src='/images/instructions-step3.svg' alt='Step 3' />
+          </Grid>
+        </Container>
+      </Box>
```

Comment:
Is it possible to format this using prettier, or would it cause conflicts with eslint?

Reply:
Formatted with prettier.